sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/SentryScheduleHook.java

```diff
@@ -13,16 +13,18 @@
 @ApiStatus.Experimental
 public final class SentryScheduleHook implements Function<Runnable, Runnable> {
   @Override
+  @SuppressWarnings("deprecation")
   public Runnable apply(final @NotNull Runnable runnable) {
-    final IHub newHub = Sentry.getCurrentHub().clone();
+    // TODO fork instead
+    final IScopes newHub = Sentry.getCurrentScopes().clone();
```

Comment:
To be renamed, once forking is done

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366